api/projects/tests/test_permissions.py

```diff
@@ -371,3 +372,29 @@ def test_regular_user_has_no_destroy_permission(self):
 
         # Then - exception thrown
         assert not result
+
+
+@pytest.mark.django_db
+def test_free_plan_has_only_fixed_projects_permission():
+    organisation = Organisation.objects.create(name="Test organisation")
+
+    user = FFAdminUser.objects.create(email="admin@test.com")
+    user_permission_group = UserPermissionGroup.objects.create(
+        name="Users", organisation=organisation
+    )
+    user_permission_group.users.add(user)
+    user.add_organisation(organisation, OrganisationRole.ADMIN)
+
+    project_permissions = ProjectPermissions()
+
+    mock_view.action = "create"
+    mock_view.detail = False
+    mock_request.data = {"name": "Test", "organisation": organisation.id}
+    mock_request.user = user
```

Comment:
I appreciate that you're following an existing pattern here, but I feel like this is going to cause us problems in the future. We're patching things onto global variables which won't be cleared before running other tests. It might result in unexpected failures depending on the order tests are run in for example. I think we should just instantiate the mocks in this individual test (and, for another PR, in the `setUp` method on the above test class perhaps). It also shouldn't add any more lines of code, because we can just pass the attributes to the MagicMock constructor.

Reply:
Yes, makes sense. I can create a separate issue for this refactoring.